### DISCUSSION TITLE
task list - allow 2 boxes

### DISCUSSION THREAD

**Discussion by siff-duke** [community/community#4534] [Projects and Issues]
It would be great if you're able to allow multiple boxes to render within the same line e.g. 

```
- [ ] [ ] some work to do
```

First box indicates that the work is completed.
Second box indicates that the work has been reviewed.

**Comment by r002** [comment]
Hi @siff-duke!  Your feedback intrigued me!  I'm not associated with GitHub at all (I'm just an enthusiastic fan! 😄) but today I had some free time so I hacked together a quick Chromium Extension that will perform custom parsing of a GitHub `issue` and then render it accordingly.  I've subscribed to this `GitHub feedback` discussion board for the past month and have seen a ton of great ideas.  Figured it'd be a fun weekend project that follows a long, proud tradition of [GitHub extensions](https://github.com/collections/github-browser-extensions).  Fiddlers gotta fiddle!  🙂

Anyway, so I hacked together this quick [POC](https://github.com/studydash/chromium-ext/blob/67b8eda1143c1589fcbba406ea8e8d89e2723d18/content-scripts/content.js#L5-L19) of your `double checkboxes` idea and have it working on my machine:

> What it looks like `without` the Chromium Extension running:

![image](https://user-images.githubusercontent.com/45280066/125136866-9ce96a80-e0d9-11eb-9ece-d3554fada77e.png)

> What it looks like `with` the Chromium Extension running:

![image](https://user-images.githubusercontent.com/45280066/125136910-ae327700-e0d9-11eb-9ceb-9f768efcd340.png)

But now my question to you:  How in your workflow do you indicate that an issue is `completed`?  With a label?  And likewise, how do you indicate that work has been `reviewed`?  Also with a label?

Thanks!  Looking forward to hacking this together some more!  ✨🚀😀

**Reply by siff-duke** [reply]
Just by convention for us. The first checkbox means that the work is completed. The second means that it has been reviewed.
